classes/Product.php

```diff
@@ -5194,10 +5188,10 @@ public static function addCustomizationPrice(&$products, &$customized_datas)
                 $product_update['customizationQuantityReturned'] = $customization_quantity_returned;
 
                 if ($customization_quantity) {
-                    $product_update['total_wt'] = $price_wt * ($product_quantity - $customization_quantity);
-                    $product_update['total_customization_wt'] = $price_wt * $customization_quantity;
-                    $product_update['total'] = $price * ($product_quantity - $customization_quantity);
-                    $product_update['total_customization'] = $price * $customization_quantity;
+                    $product_update['total_wt'] = $product_update['unit_price_tax_incl'] * $product_quantity ;
+                    $product_update['total_customization_wt'] = $product_update['unit_price_tax_incl'] * $customization_quantity;
+                    $product_update['total'] = $product_update['unit_price_tax_excl'] * $product_quantity;
+                    $product_update['total_customization'] = $product_update['unit_price_tax_excl'] * $customization_quantity;
```

Comment:
$product_update['total_customization'] is not used anywhere in the PrestaShop. I suggest to remove this variable.
in fact $product_update['total'] =  $product_update['total_customization'] ...